### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by DitchingIt** [obo-behavior/behavior-ontology#178] [Team Posts]
@matentzn Can I suggest a new release now, as I think we tied up enough loose ends yesterday to make this the baseline from which to explore bigger changes in the Process arm?

**Comment by matentzn** [comment]
Why is this a discussion and not an issue?  I made a release: https://github.com/obo-behavior/behavior-ontology/releases/tag/v2023-05-02

**Reply by DitchingIt** [reply]
@matentzn Thanks for the release. 

I thought this was more of an 'ownership' thing, and you previously suggested that you set up the team to have that proxy role (community ownership). There is no specific content to discuss as with an Issue; it is more the idea that we are at some kind of new starting point. If an Issue is still the better location, then I'm not sure of the role of a Team Post in our case (GitHub shifted them to the Discussions tab recently which is why it is now found there). As for exploring 'bigger changes to the process arm' content, I was planning to use an Issue. Can you enlighten me more re: why it should have been an Issue as I want to do it right in future? Thanks

**Reply by matentzn** [reply]
There is no right or wrong here: issues are basically requests to do something, usually fix a bug, add a term, make a release and so on. Discussion are more hazy like questions where we need to get some input on, vague ideas that are not attached to some request etc. Generally, for small ontologies, it's mostly better to just use issues, as people rarely check out the discussions.
